packages/salesforcedx-vscode-apex/src/commands/forceApexTestSuite.ts

```diff
@@ -120,7 +121,15 @@ export class TestSuiteCreator
     CancelResponse | ContinueResponse<ApexTestSuiteOptions>
   > {
     const testSuiteInput = {
-      prompt: 'Enter desired Apex test suite name:'
+      prompt: nls.localize('force_apex_test_suite_prompt_test'),
+      validateInput: value => {
+        try {
+          CreateUtil.checkInputs(value);
+        } catch (error) {
+          return error.message;
```

Comment:
note: this error message will not be localized (correct?). Can you create a ticket to ensure we localize these error message eventually?

Reply:
The error thrown from the `checkInputs` method has been localized if that's what you're asking.